gocd/bci_repo_publish.py

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/python3
+# (c) 2023 fvogt@suse.de
+# GPLv3-only
```

Comment:
recommended is `GPL-2.0-or-later` according to https://opensource.suse.com/legal/policy

Reply:
Will change it